README.md

```diff
@@ -1,13 +1,13 @@
-# Project React Native App 📱
+# Project Dont Drop Your Phone 📱
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+This app was built with the purpose of practice to use React Native and Styled Components, with the focus point of using the Accelerometer. It's a simple game where you have to shake your phone as hard and long as possible to gain points.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+The project was built by using Technigo's template "react-native-app" and the brach "shake-sensor" where some code for the accelerometer already was pre-written and packages ready to install. I also installed expo and other dependencies. I worked in the SensorComponent and started with the different shake alerts. I tried to do some better logic for them but I couldn't get that to work properly and was running out of time, so I did a simpler but less satisfying solution. Then I made the score counter. I ran in some problems to get it to work properly but I got some great help from a team mate to solve it. The plan was to give the user more points for when higher thresholds were reached but as for now, there's only points for the highest threshold. I might come back to fix this later when I have more time. I ran into some errors and needed to debug things, and I found most of the solutions by Googling, or look at how others solved their problems in Slack.
+
+If I had more time I would like to add a home screen with a logo, short rules and a start button. I tried to do that but had some problems with installing and running the packages needed, so I have to come back to it later. I also would like to have a high score list.
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+This is the project on Expo https://expo.io/@erry/projects/shake-it The Expo Go app is required to scan the QR code and open the project.
```

Comment:
Did you connect your updated project to this link?? because I still have a prompt about the old version, when I try to open it?

Reply:
Yes, I re-deployed it to see if that helped. I don't get an error on this and didn't for the first deployment either?

Reply:
This time I'm sure it's correctly updated :)